index.php

```
@@ -81,12 +81,39 @@
+    <link rel="me" href="https://www.instagram.com/djzeneyer/">
+    <link rel="me" href="https://soundcloud.com/djzeneyer">
+    <style>#wpadminbar { display: none !important; } html { margin-top: 0 !important; }</style>
+';
+        $wp_head_content .= $me_links;
+
+        ob_start();
+        wp_footer();
+        $wp_footer_content = ob_get_clean();
+
+        // Inject wp_head just before </head>
+        $html_content = str_replace('</head>', $wp_head_content . "\n</head>", $html_content);
```

PHPStan (2.1.54)

[warning] Parameter `#3` $subject of function str_replace expects array<string>|string, string|false given.


(argument.type)